wpengine/faustjs
# Use stable server actions in app router example

## Tasks


## Description

This PR introduces the new stable server actions syntax for the app router example project.



## Related Issue(s):



## Testing

1. Checkout the branch
2. `npm install` at the root
3. `npm run dev -w @faustwp/app-router-example` to fire up the example project
4. Login with different scenarios (bad username, bad password, good credentials, etc.) and notice login works properly and error messages are displayed when needed



## Screenshots



## Documentation Changes



## Dependant PRs
